src/main/java/io/github/thebusybiscuit/slimefun4/api/events/AutoDisenchanterComputeOutputEvent.java

```diff
@@ -0,0 +1,178 @@
+package io.github.thebusybiscuit.slimefun4.api.events;
+
+import javax.annotation.Nonnull;
+import javax.annotation.ParametersAreNonnullByDefault;
+
+import org.apache.commons.lang.Validate;
+import org.bukkit.event.Event;
+import org.bukkit.event.HandlerList;
+import org.bukkit.inventory.ItemStack;
+
+import io.github.thebusybiscuit.slimefun4.implementation.items.electric.machines.enchanting.AutoDisenchanter;
+
+import me.mrCookieSlime.Slimefun.api.inventory.BlockMenu;
+
+/**
+ * An {@link Event} that is fired just after {@link AutoDisenchantEvent} iff it was not cancelled.
```

Comment:
```suggestion
 * An {@link Event} that is fired just after {@link AutoDisenchantEvent} if it was not cancelled.
```

Reply:
This was intended as iff means "if and on if"

Reply:
ah okay didnt know that

Reply:
I have never seen somebody use iff, Id just go for if here

Reply:
If alone is too ambigious, so I'll go with if and only if